### ISSUE TITLE
never_update remains false after running force_update_stylesheets

### ISSUE THREAD

**Issue by melch** [sass/sass#176]
Ran into a problem where our deploy was taking forever to run through a cache reload. Turns out sass was recompiling stylesheets, even though the environment had set `Sass::Plugin.options[:never_update] = true`

Looks like this was the culprit, old_options pointed to options, which was overwritten, so the previous variable settings were never reinstated.

``` ruby
def force_update_stylesheets(individual_files = [])
  old_options = options
  self.options = options.dup
  options[:never_update] = false
  options[:always_update] = true
  options[:cache] = false
  update_stylesheets(individual_files)
ensure
  self.options = old_options
end
```

Changed line 104 and deleted 105, looks like it cleaned up the problem.

``` ruby
old_options = options.dup
```

**Comment by jpterry** [comment]
I've submitted a pull request with a fix here: https://github.com/nex3/sass/pull/175

**Comment by nex3** [comment]
Closed by cc73198e29547c170aa61365456597c6846a3f5e
